Helper.java

```diff
@@ -28,7 +30,11 @@ static void enterData() throws InvalidAnimalName {
                     Menu:\s
                     1 - Add an object
                     2 - Display all objects
-                    3 - Exit the program""");
+                    3 - Display all dog instances
```

Comment:
Lets merge 3-5 options to 1 where you can select inctance to be filtered in sub-menu